geode-redis/src/main/java/org/apache/geode/redis/internal/executor/set/RedisSet.java

```diff
@@ -79,14 +79,18 @@ public static long sadd(Region<ByteArrayWrapper, RedisSet> region,
     }
   }
 
-  public static long srem(Region<ByteArrayWrapper, RedisSet> region,
+  public static Long srem(Region<ByteArrayWrapper, RedisSet> region,
```

Comment:
The old code that returned "long" was okay wasn't it? Any reason to change this to Long?

Reply:
no good reason...reverted.